api/types.proto

```diff
@@ -624,6 +624,14 @@ message RaftConfig {
 	uint32 election_tick = 5;
 }
 
+
+// SecretsConfig specifies cluster-level secrets settings.
+message SecretsConfig {
+	// MaxSecrets specifies how many user-specified secrets there can be.  A value
+	// of 0 will be interpreted as the default maximum value.
+	uint32 max_secrets = 1;
```

Comment:
Oh, wait, spoke too soon. The proto indexes do start at 1. Could you elaborate on your question?